lib/api/settings.rb

```diff
@@ -5,4 +5,9 @@ module Api
     o.add_source!(Rails.root.join("config/api.yml").to_s)
     o.load!
   end
+
+  def self.model_to_collection(klass)
+    klass = klass.name if klass.kind_of?(Class)
+    Api::Settings.collections.to_hash.select { |_name, cfg| cfg[:klass] == klass }.keys.first
```

Comment:
Config::Options should include `Enumerable`, so `to_hash` shouldn't be necessary - also could `detect` be used over `select`?